bindings/rust/bench/README.md

```diff
@@ -0,0 +1,43 @@
+# Benchmarking s2n-tls
+
+We benchmark s2n-tls against two other common TLS libraries, Rustls and OpenSSL. We use Rust bindings for s2n-tls and OpenSSL and Criterion to benchmark these libraries. 
```

Comment:
I think this paragraph could be more concise. Probably the important information is that we use Criterion as our benchmarking suite and we use it to compare s2n-tls, Rustls, and OpenSSL.

Reply:
I moved the fact that we use rust bindings to implementation details, because I want to be transparent about that. I also kept the qualification of Rustls and OpenSSL as "commonly used TLS libraries" to clearly justify why we chose those two libraries.